app/src/app/api/v0/notification/route.ts

```diff
@@ -0,0 +1,39 @@
+import { sendEmail } from "@/lib/email";
+import AdminNotificationTemplate from "@/lib/emails/AdminNotificationTemplate";
+import ForgotPasswordTemplate from "@/lib/emails/ForgotPasswordTemplate";
+import { apiHandler } from "@/util/api";
+import { render } from "@react-email/components";
+import createHttpError from "http-errors";
+import { NextRequest, NextResponse } from "next/server";
+
+export const POST = apiHandler(
+  async (req: NextRequest, { params, session }) => {
+    const body = await req.json();
+
+    const sendNotification = await sendEmail({
```

Comment:
This should just be a static function in e.g. UserService (or a new NotificationService file rather), and not be calleable from the frontend at all!